alcs-frontend/src/app/shared/details-header/details-header.component.ts

```diff
@@ -1,27 +1,35 @@
 import { Component, Input } from '@angular/core';
 import { Router } from '@angular/router';
 import { Subject } from 'rxjs';
+import { ApplicationTypeDto } from '../../services/application/application-code.dto';
 import { ApplicationModificationDto } from '../../services/application/application-modification/application-modification.dto';
 import { ApplicationReconsiderationDto } from '../../services/application/application-reconsideration/application-reconsideration.dto';
 import { ApplicationDto } from '../../services/application/application.dto';
 import { CardDto } from '../../services/card/card.dto';
 import { CommissionerApplicationDto } from '../../services/commissioner/commissioner.dto';
+import { NoticeOfIntentDto } from '../../services/notice-of-intent/notice-of-intent.dto';
 import { MODIFICATION_TYPE_LABEL, RECON_TYPE_LABEL } from '../application-type-pill/application-type-pill.constants';
 
 @Component({
-  selector: 'app-application-header[application]',
-  templateUrl: './application-header.component.html',
-  styleUrls: ['./application-header.component.scss'],
+  selector: 'app-details-header[application]',
+  templateUrl: './details-header.component.html',
+  styleUrls: ['./details-header.component.scss'],
 })
-export class ApplicationHeaderComponent {
+export class DetailsHeaderComponent {
   destroy = new Subject<void>();
 
   linkedCards: (CardDto & { displayName: string })[] = [];
 
-  _application: ApplicationDto | CommissionerApplicationDto | undefined;
-  @Input() set application(application: ApplicationDto | CommissionerApplicationDto | undefined) {
+  @Input() heading = 'Title Here';
```

Comment:
Is it also a TODO?

Reply:
Nope, this is an obvious placeholder so that you don't forget to set the input.